hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -92,9 +93,10 @@ const (
 	TokenSecretIgnitionReachedAnnotation      = "hypershift.openshift.io/ignition-reached"
 	TokenSecretNodePoolUpgradeType            = "hypershift.openshift.io/node-pool-upgrade-type"
 
-	tuningConfigKey                = "tuning"
-	tuningConfigMapLabel           = "hypershift.openshift.io/tuned-config"
-	nodeTuningGeneratedConfigLabel = "hypershift.openshift.io/nto-generated-machine-config"
+	tuningConfigKey                  = "tuning"
+	tunedConfigMapLabel              = "hypershift.openshift.io/tuned-config"
+	nodeTuningGeneratedConfigLabel   = "hypershift.openshift.io/nto-generated-machine-config"
```

Comment:
just to clarify no need to change but do we really need two different labels here? seems having a common tuning-config would be enough to filter out to reconciliation, correct?

Reply:
PerformaceProfileController is a differenciate controller inside the NTO.

`hypershift.openshift.io/performanceprofile-config` label is used in the predicate functions to filter out ConfigMaps while `hypershift.openshift.io/tuned-config` is used to mark the MachineConfigs generated by PerformanceProfileController and Tuned controller.
These are used for different purposes so I do not think they could be merged into only one label without changing the design of NTO handling for Hypershift.